### DISCUSSION TITLE
Newly-added workflow template not visible

### DISCUSSION THREAD

**Discussion by krisalyssa** [community/community#26562] [Enterprise]
<p>We created <code>.github/workflow_templates</code> in <code>master</code> seven hours ago, but when we try to add the workflow to another repo, it’s not showing up. (We’re missing the whole “Workflows created by (organization)” section, but that’s understandable as this is our first workflow template.)</p>
<p>Is there a refresh of the organization workflow templates on a schedule instead of on push to <code>master</code> in our <code>.github</code> repo? If so, up to how long do we need to wait to see the new workflow template?</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/CraigCottingham">@CraigCottingham</a>,</p>
<p>You seem are using a wrong folder name “<strong>workflow_templates</strong>”. Change it to “<strong>workflow-templates</strong>” should work.</p>
<p>To view more details, you can see “<a href="https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template" rel="noopener nofollow ugc"><strong>Creating a workflow template</strong></a>”.</p>

**Comment by krisalyssa** [comment]
<p>Sorry, I mistyped the name of the folder in my question. We have <code>workflow-templates</code> in our repo.</p>
<p>Upon further reading of the documentation, we’re thinking that maybe the template isn’t available because our organization has a GitHub Team subscription, and the repo we’re trying to add the template to is private. Can you confirm that templates aren’t available in that case?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/CraigCottingham">@CraigCottingham</a>,</p>
<p>As the introduction from the docs about <a href="https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template" rel="noopener nofollow ugc"><strong>creating a workflow template</strong></a> mentioned,</p>
<blockquote>
<p><strong>Workflow templates can be used to create new workflows in an organizations’ public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise or GitHub One plan.</strong></p>
</blockquote>
<p>To use workflow templates in the private repositories, the organization must be part of an enterprise or GitHub One plan. For other plans, the workflow templates are not available to private repositories currently.</p>
<p>So,</p>
<blockquote>
<p>Can you confirm that templates aren’t available in that case?</p>
</blockquote>
<p>Yes, the workflow templates are not available in your case.</p>

**Comment by karlbaumhauer** [comment]
<p>hey <a class="mention" href="/brightran">@brightran</a>,</p>
<p>I encountered the same issue as <a class="mention" href="/CraigCottingham">@CraigCottingham</a>. The whole “Workflows created by (organization)” section is missing.</p>
<p>We are using an enterprise plan and therefore it should be available for private repos as well?!?</p>
<p>Any idea what I am missing here?</p>

**Comment by karlbaumhauer** [comment]
<p>Was able to fix it with the enterprise support. Issue/Solution: Base repo (.github) needs to be public. They plan to provide it for internal repos as well but the current status needs a public repo.</p>

**Comment by lenny-dou** [comment]
<p>Any links to issues when workflow-templates will be available for internal private Github Enterprise Repos?  Is there an expected ETA?</p>

**Comment by AlexDCraig** [comment]
<p>Running into this issue as well on internal repos with GHE. Would love to have this feature.</p>

**Comment by mickeygousset** [comment]
<p>Looks like it is on the roadmap for Q3 of 2021 (Jul-Sept).  <a href="https://github.com/github/roadmap/issues/98" class="inline-onebox">Actions: Centrally managed workflow templates · Issue #98 · github/roadmap · GitHub</a></p>
